element_miniscope/imaging.py

```diff
@@ -64,7 +64,7 @@ def get_imaging_root_data_dir() -> str:
 @schema
 class ProcessingMethod(dj.Lookup):
     definition = """
-    processing_method: char(24)
+    processing_method: char(32)
     ---
     processing_method_desc: varchar(1000)
     """
```

Comment:
For lookup contents, 
1. do we still want to include caiman and suite2p here? I feel that we could delete since this is miniscope element.
2. Also, 'miniscope_analysis' seems to be a too general name. We are likely to support multiple (more than one) processing methods for miniscope.

Reply:
Good points.
1. I removed Suite2p, and kept CaImAn because we might end up supporting the CaImAn package adapted for miniscope analysis.
2. Agreed.  The package name is `MiniscopeAnalysis`, but we can store it in the table as `mcgill_miniscope_analysis`.